wp-content/themes/k1-theme/templates/page-getstarted.php

```diff
@@ -0,0 +1,103 @@
+				<div class="row d-flex justify-content-center">
+					<div class="col-8">
+						<h2 class="headline"><span class="text-white">what are your ministry's greatest needs?</span></h1>
+						<h4>Tell Us About Your Organization, And We'll Connect You With  Ministry Support As Soon As Possible.</h4>
+                    </div>
+				</div>
+			</div>
+		</section>
+
+        <aside class="quotes row">
+		<?php $content->get_color_background_layers( 'quotes', 'zig-zag-right', array( 'k1-retreat-01', 'webp' ) ); ?>
+            <div class="swiper text-center" id="quotes-swiper">
+                <div class="swiper-wrapper">
+                    <div class="swiper-slide">
+                        <img src="<?php k1_get_image_asset_url('quoteimg', 'png')?>"><br><br>
+                        <p class="text-white">this is the place where an awesome quote from someone that we have been able to serve would go.</p>
+                    </div>
+                    <div class="swiper-slide">
+                        <img src="<?php k1_get_image_asset_url('quoteimg', 'png')?>"><br><br>
+                        <p class="text-white">blurb testing blurb</p>
+                    </div>
+                    <div class="swiper-slide">
+                        <img src="<?php k1_get_image_asset_url('quoteimg', 'png')?>"><br><br>
+                        <p class="text-white">blurb blurb testing</p>
+                    </div>
+                    <div class="swiper-slide">
+                        <img src="<?php k1_get_image_asset_url('quoteimg', 'png')?>"><br><br>
+                        <p class="text-white">blurb blurb testing</p>
+                    </div>
+                    <div class="swiper-slide">
+                        <img src="<?php k1_get_image_asset_url('quoteimg', 'png')?>"><br><br>
+                        <p class="text-white">blurb blurb testing</p>
+                    </div>
+                    <div class="swiper-slide">
+                        <img src="<?php k1_get_image_asset_url('quoteimg', 'png')?>"><br><br>
+                        <p class="text-white">blurb blurb testing</p>
+                    </div>
+                    <div class="swiper-slide">
+                        <img src="<?php k1_get_image_asset_url('quoteimg', 'png')?>"><br><br>
+                        <p class="text-white">blurb blurb testing</p>
+                    </div>
+                </div>
+                <div class="swiper-button-prev"></div>
+                <div class="swiper-button-next"></div>
+            </div>
+        </aside>
+		<section class="">
+			<div class="container-fluid">
+				<div class="row d-flex justify-content-center align-items-center">
+					<div class="col-8">
+						<h2 class="headline text-center"><span class="">Wouldn't it be great <br> to have a coach or support system <br> without breaking the budget?</span></h1>
+                    </div>
+				</div>
+			</div>
+		</section>
+        <section class="problems">
```

Comment:
This whole section (`.problems`) already exists as a template part, so you can just delete it.